inst/data.json

```diff
@@ -1,10 +1,10 @@
 {
     "root": "https://mrcdata.dide.ic.ac.uk/mint",
-    "directory": "20210313",
+    "directory": "20210512",
     "files": {
-        "index": "data_index_Vector_tool_updated_params_uncertainty_no_zero_20210313.rds",
-        "prevalence": "data_values_Vector_tool_updated_params_uncertainty_no_zero_20210313.rds",
-        "table": "data_aggregate_Vector_tool_updated_params_uncertainty_no_zero_20210313.rds"
+        "index": "data_index_Vector_tool3_20210512.rds",
+        "prevalence": "data_values_Vector_tool3_20210512_filtered.rds",
```

Comment:
This file is the result of executing [this logic](https://github.com/mrc-ide/mintr/pull/56/files#diff-67c4f451e1e58079690e44cbd5dbe3af90b755a564275cb5020552fe71757634L52-L56) in advance, which dramatically decreases download and import times, as well as memory usage, which ensures that the GitHub Actions in particular continue to work despite the increase in data volumes as a result of new prevalence values.